Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -133,7 +138,8 @@ def row_centery(row):
     return c
 
 class Table:
-    re_month = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
+    re_month = re.compile("(january|february|march|april|may|june|july|august\
+                          |september|october|november|december)")

```

Comment:
@Evildoor,
> Commented debug messages in the code - messages like this one are left because they are used to debug frequently performed operations, so if i'll delete them every time then i'll most likely have to write them again just as often. If logging will be introduced and it would be possible to replace commented debug messages with it - great. But right now we have no better option, right?

We have an option of stashed changes in Git.
The workflow is as follows:
* make changes to the code, with all the debug messages you need
* use `git add --patch <files>` instead of `git add <files>` to avoid adding debug messages to the staged changes
* `git commit`
* `git stash` (hide local changes -- the debug messages indeed)
* `git push`
* `git stash pop` (restore local changes)

After that you have the code without debug messages in the repository, and with debug messages -- in a local copy.